ChainSafe/Multix
# Better show call data

closes #157 
- use transferKeepAlive for any balance transfer
- show the full call (previously only the method)
- explain why we have no info to show

How to test:
- Create/add a multisig on pjs/apps
- make a tx on pjs/apps and don't forget to copy the callData
- go on Multix and see the unfinalized proposal
- add the callData and verify the call

![image](https://github.com/ChainSafe/Multix/assets/33178835/2ecd6692-c761-4558-931e-f53b3f8a0b6b)
![image](https://github.com/ChainSafe/Multix/assets/33178835/c80cbbe6-4208-4870-a8b3-f97ca1861675)
![image](https://github.com/ChainSafe/Multix/assets/33178835/c4701dbe-2f89-4031-a2c1-49b0524014c3)
